### ISSUE TITLE
[GENERAL] How to make a quest that does not have any yellow mark?

### ISSUE THREAD

**Issue by BarbzYHOOL** [azerothcore/forum#31]
How to make a quest that is kinda "hidden", that does not have an exclamation mark or an interrogation mark?

**Comment by sluggor** [comment]
This sounds like a cool idea; if this weekend wasn't so busy I'd love to tinker around in the database to see if it was feasible.

I would think, though, that you might be able to attach a gossip menu to an NPC and then add a gossip option that would fire off a smartscript to initiate a quest?

**Comment by BarbzYHOOL** [comment]
Yes, there were hidden quests like that with special events and they can probably be scripted with SAI.

But if I wanted to mass hide the exclamation marks, that would be tricky.

Maybe there is a flag I don't know though?
